### ISSUE TITLE
Usability: NVDA is not reciting the buttons and contents for "Explorer" view option.

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/vscode#17226]
VS Code Version: 1.6.1
OS Version:1607
NVDA version:-2016.2.1

**Steps to Reproduce:**
Prerequisite: NVDA need to be opened.
1.Open visual studio code.
2.Using "Ctrl+ Shift+ E", key navigate to the Explorer button.
3.Using "Tab" ,key navigate to the side bar areas.

**Expected Result:**
NVDA should recite the buttons and contents present in the side bar areas for "Explorer" view bar option.

**Actual Result:**
NVDA is not reciting anything for the contents and buttons present in the side bar areas.

**User Impact:**
Screen Reader user doesn't get to know the purpose of the control as it's not getting announced.

![inspectinformation_visualstudiocodeeditor_sidebararea](https://cloud.githubusercontent.com/assets/22956130/21212415/73ba7590-c2b2-11e6-8664-065375a94ef8.jpg)
![narratorbuddyinformation_vscodeeditor_sidebararea](https://cloud.githubusercontent.com/assets/22956130/21212418/775e0216-c2b2-11e6-947e-03dcec36552d.jpg)
![nvda_speechviewerinformation_vscodeeditor_sidebararea](https://cloud.githubusercontent.com/assets/22956130/21212419/79c77064-c2b2-11e6-8c15-66dd21cd11e6.jpg)

**Comment by bpasero** [comment]
@WiproPSAccTest I cannot reproduce, does it reproduce with 1.8?

**Comment by WiproPSAccTest** [comment]
@bpasero This  issue is not reproduce with 1.8.1.
